diff --git a/node_modules/nativewind/node_modules/react-native-css-interop/.cache/android.js b/node_modules/nativewind/node_modules/react-native-css-interop/.cache/android.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/nativewind/node_modules/react-native-css-interop/.cache/ios.js b/node_modules/nativewind/node_modules/react-native-css-interop/.cache/ios.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/nativewind/node_modules/react-native-css-interop/.cache/macos.js b/node_modules/nativewind/node_modules/react-native-css-interop/.cache/macos.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/nativewind/node_modules/react-native-css-interop/.cache/native.js b/node_modules/nativewind/node_modules/react-native-css-interop/.cache/native.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/nativewind/node_modules/react-native-css-interop/.cache/windows.js b/node_modules/nativewind/node_modules/react-native-css-interop/.cache/windows.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/nativewind/node_modules/react-native-css-interop/dist/runtime/components.js b/node_modules/nativewind/node_modules/react-native-css-interop/dist/runtime/components.js
index 8835a2d..7bb1ae4 100644
--- a/node_modules/nativewind/node_modules/react-native-css-interop/dist/runtime/components.js
+++ b/node_modules/nativewind/node_modules/react-native-css-interop/dist/runtime/components.js
@@ -5,7 +5,6 @@ const react_native_1 = require("react-native");
 const api_1 = require("./api");
 (0, api_1.cssInterop)(react_native_1.Image, { className: "style" });
 (0, api_1.cssInterop)(react_native_1.Pressable, { className: "style" });
-(0, api_1.cssInterop)(react_native_1.SafeAreaView, { className: "style" });
 (0, api_1.cssInterop)(react_native_1.Switch, { className: "style" });
 (0, api_1.cssInterop)(react_native_1.Text, { className: "style" });
 (0, api_1.cssInterop)(react_native_1.TouchableHighlight, { className: "style" });
